spark-indexer/src/test/java/org/uniprot/store/spark/indexer/suggest/SuggestDocumentsToHPSWriterTest.java

```diff
@@ -450,6 +450,30 @@ void getUniParcTaxonomy() {
                 () -> assertTrue(resultMap.containsKey("10066")));
     }
 
+    @Test
+    void testGet559292() {
+        SuggestDocumentsToHPSWriter writer = new SuggestDocumentsToHPSWriter(parameter);
+
+        JavaRDD<SuggestDocument> suggestRdd =
+                writer.getUniProtKbOrganism(
+                        flatFileRDD,
+                        new TaxonomyRDDReaderFake(parameter, true, true).loadTaxonomyLineage());
+        assertNotNull(suggestRdd);
+        List<SuggestDocument> suggests = suggestRdd.collect();
+        assertEquals(91L, suggests.size());
+        SuggestDocument document =
+                suggests.stream()
+                        .filter(doc -> doc.id.equals("559292"))
+                        .findFirst()
+                        .orElseThrow(AssertionFailedError::new);
+        assertNotNull(document);
+        assertEquals(ORGANISM.name(), document.dictionary);
+        assertEquals("559292", document.id);
+        assertTrue(document.altValues.contains("Baker’s yeast"));
+        assertTrue(document.altValues.contains("Baker’s yeast"));
```

Comment:
Duplicated lines 471 and 472, can you please remove one, or change the description of one of them